### PR TITLE
Linter: Recover panics

### PR DESCRIPTION
**Pull request by julienduchesne**
Currently, the linter is useful but panics on a few projects. The panics make it unusable on a large set of projects

With this PR, it will fail linting but all projects will still get linted

### CODE REVIEW

File: pkg/jsonnet/lint.go [hunk 1]
```diff
@@ -37,42 +44,21 @@ func Lint(fds []string, opts *LintOpts) error {
...
-			resultCh <- result{failed: failed, output: buf.String()}
-			log.Debug().Str("file", file).Dur("duration_ms", time.Since(startTime)).Msg("linted file")
+			buf, success := lintWithRecover(file)
+			resultCh <- result{success: success, output: buf.String()}
 		}
 	}
 
+	if opts.Parallelism <= 0 {
+		opts.Parallelism = 4
+	}
```
Comment (jvrplmlmn) [inline pkg/jsonnet/lint.go:61]: Considering that we are using the same default as when one is not specified, should we check this in `cmd/tk/lint.go`? Or alternatively check at the beginning of the function an error?

**Review by jvrplmlmn — APPROVED** [review]
LG2M modulo a small comment 👍

File: pkg/jsonnet/lint.go (hunk 1 above)
Comment (julienduchesne) [reply pkg/jsonnet/lint.go:61]: True, it has a default already in `cmd`. I converted it to an error instead

Comment (jvrplmlmn) [reply pkg/jsonnet/lint.go]: 👍

**Pull request merged**